.github/workflows/production_pipeline.yml

```diff
@@ -0,0 +1,185 @@
+name: 'Production CI/CD Pipeline'
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+
+concurrency: 'production'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13.5
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: postgres
+          POSTGRES_DB: data_collector
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        # tmpfs makes DB faster by using RAM
+        options: >-
+          --mount type=tmpfs,destination=/var/lib/postgresql/data
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    env:
+      RAILS_ENV: test
+      GEMFILE_RUBY_VERSION: 3.1.0
+      DB_HOST: localhost
+      DB_DATABASE: data_collector
+      DB_USERNAME: postgres
+      DB_PASSWORD: password
+
+    steps:
+      - name: Get latest release with tag
+        id: latestrelease
+        run: |
+          echo "::set-output name=releasetag::$(curl -s https://api.github.com/repos/weide-zhou/ticket13/releases/latest | jq '.tag_name' | sed 's/\"//g')"
+      - name: Confirm release tag
```

Comment:
Should we use instead https://docs.github.com/en/rest/reference/releases#get-the-latest-release ?

Reply:
Work checking https://docs.github.com/en/actions/learn-github-actions/environment-variables as well

Reply:
We are using that first one - I just had the wrong repo in there